lib/phoenix_live_view/html_tokenizer.ex

```diff
@@ -43,19 +43,19 @@ defmodule Phoenix.LiveView.HTMLTokenizer do
   end
 
   @doc """
-  Tokenize the given text according tho the given params.
+  Tokenize the given text according to the given params.
```

Comment:
I have read this so many times and I couldn't get any of these typos 🙈 . Thank you ❤️